CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 
 ## next
+## 1.0.0-rfc.2
```

Comment:
Aucune importance mais release candidate c'est "rc" habituellement non ? "rfc" request for comment, mais c'est un doc pas une version.
Je teste la v1 sur mesaidesvelo rapidement !

Reply:
Oups, effectivement, j'ai confondu les deux :p